lighthouse-cli/index.js

```diff
@@ -91,6 +96,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('mobile', true)
   .default('output', Printer.OUTPUT_MODE.pretty)
   .default('output-path', 'stdout')
+  .default('launch-chrome', true)
```

Comment:
yeah. If it's declared boolean, by default it'll be false rather than undefined.